js/fixToc.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Double quotes are being used, should be single. Please refer to papermoon style guide to properly format this